packages/file_selector/file_selector/example/lib/save_text_page.dart

```diff
@@ -22,8 +22,9 @@ class SaveTextPage extends StatelessWidget {
     // This demonstrates using an initial directory for the prompt, which should
     // only be done in cases where the application can likely predict where the
     // file will be saved. In most cases, this parameter should not be provided.
-    final String initialDirectory =
-        (await getApplicationDocumentsDirectory()).path;
+    final String? initialDirectory = !kIsWeb
```

Comment:
Optional nit: I find double negation harder to parse; I would prefer this as `kIsWeb ? null : ...`.

Reply:
Yes, I thought about this too! Let me fix it.